x/genutil/gentx_test.go

```diff
@@ -47,7 +47,7 @@ func (suite *GenTxTestSuite) SetupTest() {
 	app := simapp.Setup(checkTx)
 	suite.ctx = app.BaseApp.NewContext(checkTx, tmproto.Header{})
 	suite.app = app
-	suite.encodingConfig = simapp.MakeEncodingConfigTests()
+	suite.encodingConfig = simapp.MakeEncodingConfig()
```

Comment:
I think places like here (and some other places), we should remove the creation of a new `suite.encodingConfig`, and just use `suite.app.appCodec` or `suite.app.legacyAmino` (because those 2 latter have already registered interfaces). Maybe for another PR.

Reply:
Yes, I will make an issue for that.

Reply:
https://github.com/cosmos/cosmos-sdk/issues/7631